components/collection/drop/ItemsGrid.vue

```diff
@@ -30,6 +30,7 @@
       hide-listing
       show-timestamp
       :reset-search-query-params="['sort']"
+      fetch-onchain-data
```

Comment:
when the collection is generative art, could we add this `fetch-onchain-data` to the collection details page, too?